internal/provider/main_test.go

```diff
@@ -90,6 +90,6 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		panic(err)
 	} else {
-		m.Run()
+		os.Exit(m.Run())
```

Comment:
Why is this necessary now? We're exiting anyway?

Reply:
linter was complaining